dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/staxutils/StaxUtils.java

```diff
@@ -243,6 +243,7 @@ private static int getInteger(String prop, int def) {
             }
             return i;
         } catch (Throwable t) {
+            LOG.finest("getInteger: Ignoring exception: " + t);  // Liberty Change
```

Comment:
Same for this file. I think its fine to have the added trace logging, but make sure the trace injection is turned off for this class.

Reply:
Disabled trace injection for StaxUtils.class.